extensions/wikia/PhalanxII/templates/PhalanxStatsSpecial_index.php

```diff
@@ -1,10 +1,41 @@
-<?= $table ?>
-
-<? if ( isset( $editUrl ) ): ?>
-<a href="<?= htmlspecialchars($editUrl) ?>" class="modify"><?= wfMsg('phalanx-link-modify') ?></a>
-<? endif ?>
+<table class="wikitable phalanx-stats-table">
+	<thead>
+		<tr>
+			<th><?= wfMessage( 'phalanx-stats-table-id' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-user' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-type' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-create' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-expire' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-exact' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-regex' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-case' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-language')->escaped(); ?></th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+		<?php foreach ( $firstRow as $entry ): ?>
+			<td><?= htmlspecialchars( $entry ); ?></td>
+		<?php endforeach; ?>
+		</tr>
+		<tr>
+			<th><?= wfMessage( 'phalanx-stats-table-text' )->escaped(); ?></th>
+			<td colspan="8"><?= htmlspecialchars( $text ); ?></td>
+		</tr>
+		<tr>
+			<th><?= wfMessage( 'phalanx-stats-table-reason' )->escaped(); ?></th>
+			<td colspan="8"><?= $reason ?></td>
```

Comment:
Yes after Grunny's note I decided to feed it to the `Parser` since it contains wikitext anyways (we commonly put `[[w:Help:Spam|spam]]` in there)